telluric/georaster.py

```diff
@@ -736,12 +742,20 @@ def empty_from_roi(cls, roi=None, resolution=None,
     def _cleanup(self):
         for f in self._opened_files:
             f.close()
```

Comment:
I would pop closed element each time a close action succed instead to wait the end of all close to clean  self._opened_files this could avoid to leave dirty results in case of "in the middle" errors

Reply:
Thanks @luipir !